agi_hoheitsgrenzen_pub_sogis/build.gradle

```diff
@@ -0,0 +1,17 @@
+import ch.so.agi.gretl.tasks.*
+import ch.so.agi.gretl.api.TransferSet
+
```

Comment:
Schreib hier ebenfalls einen Kommentar wie in https://github.com/sogis/gretljobs/blob/502c5e6094ee61d3d31e210bdf467b9cd9f517bd/arp_nutzungsvereinbarung_pub_sogis/build.gradle#L6 rein. Das finde ich dort gut so, weil Source und Target DB die gleiche ist.